### ISSUE TITLE
Prime - AggregateError: All promises were rejected

### ISSUE THREAD

**Issue by kennymc-c** [vogler/free-games-claimer#126]
Similar to #125, according to the log I get an error message when claiming, although some games have been claimed according to the account. Terraformers seems to be the problem here and it is the only game from this weeks 4 games that is not claimed. Maybe due to the missing/+18 age rating like Epic #120?

```
2023-04-20 16:11:20.925 started checking prime-gaming
Signed in as kennymc.c
Number of already claimed games (total): 11
Number of free unclaimed games (Prime Gaming): 0
Number of free unclaimed games (external stores): 4
Current free game: Metal Slug 4
[AggregateError: All promises were rejected] {
  [errors]: [
    page.click: Timeout 60000ms exceeded.
    =========================== logs ===========================
    waiting for locator('button:has-text("Claim now")')
    ============================================================
        at file:///fgc/prime-gaming.js:129:29 {
      name: 'TimeoutError'
    },
    page.click: Timeout 60000ms exceeded.
    =========================== logs ===========================
    waiting for locator('button:has-text("Complete Claim")')
    ============================================================
        at file:///fgc/prime-gaming.js:129:73 {
      name: 'TimeoutError'
    },
    page.waitForSelector: Timeout 60000ms exceeded.
    =========================== logs ===========================
    waiting for locator('div:has-text("Link game account")') to be visible
    ============================================================
        at file:///fgc/prime-gaming.js:129:122 {
      name: 'TimeoutError'
    }
  ]
}
```

**Comment by brockm218** [comment]
+1, having the same issue, appears to be a very recent change. Will update after work if I have a chance to investigate the root cause.

**Comment by kennymc-c** [comment]
Interesting that Ninja Masters's was not claimed in your case. I ran the script a total of 3 times, the last time with VNC connected. But the last time nothing was claimed. Nothing happened until the timeout.

**Comment by kennymc-c** [comment]
Can confirm, with the fourth run i also got Terraformers

**Comment by vogler** [comment]
Yes, the buttons are now exactly the same (both `data-a-target="ExternalOfferClaim"`).
![image](https://user-images.githubusercontent.com/493741/233502373-2c4ae025-71e8-4812-b98a-bcad08cf0a20.png)
@OJ7 I don't have a `class="item-how-to-play"`, but there are several possible ways to keep the internal/external cards apart:
![image](https://user-images.githubusercontent.com/493741/233502340-8924c7e6-b448-4aa4-b6c4-d28c38c691b0.png)

- internal: `data-a-target="claim-prime-offer-card"`, no `a`
- external: `data-a-target="learn-more-card"`, `a` with `href="/loot/...`

**Comment by kennymc-c** [comment]
This week 3 games could be claimed in one run. Looking for Aliens appeared a little later and was not yet available at that time but could be claimed later

**Comment by vogler** [comment]
Looks like they changed it back and the button texts are the same as before - `Claim game` for internal and `Claim` for external games.

I'll close this - comment if there's still some issue.

**Comment by destrugter** [comment]
I love this app so much! I found this issue and I'm currently experiencing this bug. They all say "claim" and it only claims one game and then fails to claim the rest on Prime Gaming

**Comment by vogler** [comment]
@destrugter For me it's the same as before, i.e., `Claim game` for internal games:
![image](https://user-images.githubusercontent.com/493741/236680130-aad47656-1fc6-404f-8cff-fa154c806bab.png)

**Comment by destrugter** [comment]
I'm running it through VNC and watching it, this is what it sees on my end:

![image](https://user-images.githubusercontent.com/2092138/236681125-4d710f5e-d91b-425b-82d1-9c9884f60b03.png)

**Comment by vogler** [comment]
Mh, strange that it's different. I have a German account. What region is your account in?

> * internal: `data-a-target="claim-prime-offer-card"`, no `a`
> * external: `data-a-target="learn-more-card"`, `a` with `href="/loot/...`

I can change the selectors to that since it seems to be the same for both situations.

**Comment by destrugter** [comment]
My account is USA.

That's the same text I see in the inspector as well

**Comment by vogler** [comment]
No more problems with the buttons after the fix?

**Comment by destrugter** [comment]
Yes, the last run looks perfect. Thank you so much!

**Comment by kennymc-c** [comment]
I get this error again for Neverwinter Nights. I think there has been another change for external games on the second page where it now says "Get game" instead of "Complete Claim" and the script runs into a timeout.
![Bildschirmfoto 2023-06-16 um 19 04 06](https://github.com/vogler/free-games-claimer/assets/14995555/f31c1d6d-fd94-4ae2-88bc-d00c85c9cb9f)

**Comment by vogler** [comment]
@kennymc-c This should be fixed with #158.

However, they now mixed up the internal/external games even more: [Revita](https://gaming.amazon.com/revita-fgwp-jun23/dp/amzn1.pg.item.76a7ecf7-4200-4189-910e-b3687d1f2f75) was just identified as neither because its DOM-element is like for an external game (also has its own URL), but the text was `Claim game` as for an internal game (which it actually is, i.e., Amazon Games as store).
